src/app/containers/InlineLink/index.jsx

```diff
@@ -29,6 +30,10 @@ const InlineLinkContainer = ({ locator, blocks }) => {
     );
   }
 
+  if (isExternal) {
+    blocks.push(<VisuallyHiddenText>, external</VisuallyHiddenText>);
```

Comment:
I dont think this is right, the blocks array contains objects of type and model https://github.com/bbc/simorgh/blob/latest/src/app/containers/Blocks/index.jsx#L30-L35

Reply:
Argh, good point!